R/render-table.R

```diff
@@ -3,33 +3,103 @@
 #' Creates a reactive table that is suitable for assigning to an \code{output}
 #' slot.
 #'
-#' The corresponding HTML output tag should be \code{div} and have the CSS class
-#' name \code{shiny-html-output}.
+#' The corresponding HTML output tag should be \code{div} and have the CSS
+#' class name \code{shiny-html-output}.
 #'
 #' @param expr An expression that returns an R object that can be used with
 #'   \code{\link[xtable]{xtable}}.
-#' @param ... Arguments to be passed through to \code{\link[xtable]{xtable}} and
-#'   \code{\link[xtable]{print.xtable}}.
+#' @param striped,condensed,hover,bordered Logicals: if \code{TRUE}, apply
+#'   the corresponding Bootstrap table format to the output table.
```

Comment:
I'm a bit ambivalent about which way to go, but I'm more on the side of having separate arguments. 

The one drawback is that having four separate arguments is a bit unwieldy, but the benefits are that:
* The arguments are easily discoverable with tab-completion.
* Conceptually the options are independent from each other.
* Non-binary values are possible for these options (which might matter for `condensed`).
* We're currently making `condensed` default to `TRUE`, and with a char vector argument, if someone wanted to keep that default and also use another option, they'd have to do something like `format = c("condensed", "bordered")`.

A real-world example of a function that takes a char vector is `install.packages`: the `dep` argument can be a subset of `c("Depends", "Imports", "LinkingTo", "Suggests", "Enhances")`. I can never remember what the choices are and I always have to look it up in the documentation.